src/ros/build-env-utils.ts

```diff
@@ -68,23 +68,27 @@ async function updateCppPropertiesInternal(): Promise<void> {
         });
     }));
 
-    let platform: string = os.platform();
-    let c_cpp_properties: any = {
+    if (process.platform === "win32") {
+        includes.push(path.join("C:", "opt", "rosdeps", "x64", "include", "**"));
+    } else {
+        includes.push(path.join("/", "usr", "include"));
+    }
+
+    // https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/c_cpp_properties.json.md
+    const cppProperties: any = {
         configurations: [
             {
-                browse: { databaseFilename: "", limitSymbolsToIncludedHeaders: true },
+                browse: {
+                    databaseFilename: "",
+                    limitSymbolsToIncludedHeaders: true,
+                },
                 includePath: [...includes],
-                name: "Windows",
+                name: "ROS",
```

Comment:
`Linux` and `Win32` are special identifiers, the `intelliSenseMode` was by default mapped to `gcc-x64` because this name field was `Linux`.

https://github.com/Microsoft/vscode-cpptools/blob/master/Documentation/LanguageServer/c_cpp_properties.json.md#name